docs/en/install.md

```diff
@@ -21,6 +21,22 @@ If mmcv and mmcv-full are both installed, there will be `ModuleNotFoundError`.
 
 ## Installation
 
+### A from-scratch setup script
+
+Assuming that you already have CUDA 10.1 installed, here is a full script for setting up MMRotate with conda.
```

Comment:
Assuming that you already have CUDA 10.1 installed, here is a full script for setting up MMRotate with conda. You can refer to the step-by-step installation instructions in the next section.